api/src/bigquerytest/java/org/pmiops/workbench/api/CohortReviewControllerBQTest.java

```diff
@@ -44,7 +44,6 @@
 import org.pmiops.workbench.db.dao.CohortReviewDao;
 import org.pmiops.workbench.db.dao.ParticipantCohortStatusDao;
 import org.pmiops.workbench.db.dao.UserDao;
-import org.pmiops.workbench.db.dao.UserRecentResourceService;
```

Comment:
lots of these changes because I moved it from `db.dao` to `workspaces.resources`